schemas/IDLs/org.gel.models.metrics.avro/1.0.1/RareDiseaseInterpretationPipeline.avdl

```diff
@@ -1,96 +0,0 @@
-@namespace("org.gel.models.metrics.avro")
-/**
-This protocol defines the gel metrics we calculate and store for bams
-*/
-protocol RDInterpretationPipeline {
-
-/*
-This is used in `File`
-Types:
-* `BAM`: alignment
-* `gVCF`: genomic VCF for variants
-* `VCF_small`: VCF file for SNV and indel
-* `VCF_somatic_small`: VCF file for somatic SNV and indel
-* `VCF_CNV`: VCF file for copy number variants
-* `VCF_somatic_CNV`: VCF file somatic for copy number variants
-* `VCF_SV`: VCF file for structural variants only
-* `VCF_somatic_SV`: VCF file for somatic structural variants only
-* `VCF_SV_CNV`: VCF file for CNV and SVs
-* `SVG`: an SVG for example of a pedigree
-* `ANN`: a Json File for the annotation file from org.opencb.biodata.models/1.2.0-SNAPSHOT/variantAnnotation.avdl
-* `BigWig`: a bigwig file with the genome coverage
-* `MD5Sum`: a MD5Sum file
-* `ROH`: a BED file with Regions of homozygosity
-* `OTHER`: other unspecified file type
-**/
-enum FileType {
-    BAM,
-    gVCF,
-    VCF_small,
-    VCF_somatic_small,
-    VCF_CNV,
-    VCF_somatic_CNV,
-    VCF_SV,
-    VCF_somatic_SV,
-    VCF_SV_CNV,
-    SVG,
-    ANN,
-    BigWig,
-    MD5Sum,
-    ROH,
-    OTHER
-    }
-
-/**
-This defines a file
-This Record is defined by the sampleID and a URI
-Currently SampleID can be a single String or an array of strings if multiple samples are associated with the same file
-**/
-record File {
-    /**
-    Unique ID(s) of the Sample, for example in a multisample vcf this would have an array of all the sample ids
-    */
-    union {null, string, array<string>} SampleId;
-
-    /**
-    URI PATH
-    */
-    string URIFile;
-
-    FileType fileType;
-
-    union {null, string} md5Sum;
-
-}
-
-enum StepStatus {failed, ready}
-
-record Step {
-    string stepName;
-    string date;
-    StepStatus status;
-
-
-}
-
-record RareDiseaseInterpretationStatus {
```

Comment:
@Antonior26 @priesgo 

The CIP API requires this in cip_connectors.py:
https://github.com/genomicsengland/interpretation/blob/64c2ca5c8175076ad95a0eeb04143d52161670ab/interpretationAPI/cip_connectors.py#L11

Is there any way we can bring this back ?